src/OrleansAWSUtils/Reminders/DynamoDBReminderTable.cs

```diff
@@ -181,6 +214,10 @@ private static ReminderEntry Resolve(Dictionary<string, AttributeValue> item)
             }
         }
 
+        /// <summary>
+        /// Aync method to clear the reminder table
+        /// </summary>
+        /// <returns>A task object </returns>
```

Comment:
Either drop the entire return, or just say `A task that completes when clearing the table completes`. Nevertheless, that is the standard way of using a task, so I would not add it.